### ISSUE TITLE
Next steps

### ISSUE THREAD

**Issue by kashizui** [kashizui/rnn-sentiment-analysis#2]
- [x] Get baseline of randomized embeddings
- [x] Vary dimensionality of internal representations
- [ ] ~~Turn off training of embedding after N epochs~~
- [x] Proper handling of unknown tokens
- [ ] Visualize loss during training
- [ ] Experiment with more pre-trained embeddings
- [ ] ~~Alternative approaches to regularization to tackle overfitting~~
- [x] Introduce stacked LSTMs
- [ ] ~~Experiment with non-binary sentiment values~~
